modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastViewModel.kt

```diff
@@ -148,7 +148,7 @@ class PodcastViewModel
                 Flowable.just(it).concatWith(podcastManager.observePodcastByUuid(it.uuid).skip(1))
             }
             .observeOn(AndroidSchedulers.mainThread())
-            .doOnNext { newPodcast ->
+            .doOnNext { newPodcast: Podcast ->
```

Comment:
This fixes the following Lint issue

<img width="951" alt="Screenshot 2024-09-13 at 9 39 40 PM" src="https://github.com/user-attachments/assets/c8b1591f-b280-4f5f-af60-cb8d6cdba46b">